source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextBlockRenderer.mm

```diff
@@ -195,6 +196,40 @@ + (NSNumber *)getTextStrokeWidthForWeight:(TextWeight)weight
             return @-2;
     }
 }
+// given string with short form of date (MM/dd/yyyy), convert it to
+// NSDateFormatterMediumStyle or NSDateFormatterLongStyle in local language
++ (void) getLocalizedDate:(std::shared_ptr<TextBlock> const &)txtBlck stringWithDate:(std::string &)dateParsedString
+{
+    std::vector<std::shared_ptr<DateTimePreparsedToken>> DateTimePreparsedTokens =  DateTimePreparser(txtBlck->GetText()).GetTextTokens();
+    for(auto section : DateTimePreparsedTokens){
+        if(section->GetFormat() != DateTimePreparsedTokenFormat::RegularString) {
+            NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+            [formatter setDateFormat:@"MM/dd/yyy"];
```

Comment:
>yyy [](start = 45, length = 3)

Three-digit years are uncommon.

Reply:
you're right

---
In reply to: [171674845](https://github.com/Microsoft/AdaptiveCards/pull/1194#discussion_r171674845) [](ancestors = 171674845)